<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>BioFuM Experiment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="Main_layout">

   <!-- Top Row -->
    <item row="0" column="0">
      <layout class="QGridLayout" name="Controls">
        <item row="0" column="0">
         <widget class="QWidget" name="Main_widget" native="true"/>
        </item>
      </layout>
    </item>

    <!-- Bottom Row -->
    <item row="1" column="0">
      <widget class="QWidget" name="Device_viewers" native="true">
        <layout class="QGridLayout" name="Device_viewers_layout">
          <item row="0" column="0">
            <widget class="QWidget" name="Camera_viewer" native="true"/>
          </item>
          <item row="0" column="1">
            <widget class="QWidget" name="Spectrometer_viewer" native="true"/>
          </item>
        </layout>
      </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
